README.md

```diff
@@ -61,73 +61,59 @@ Each comes with different trade-offs described below.
+ - minimal environment pre-requisites (no need to compile Go code)
+ - see ["hc-install is not a package manager"](https://github.com/hashicorp/hc-install#hc-install-is-not-a-package-manager)
 
-v0_14_0 := version.Must(version.NewVersion("0.14.0"))
+### Installation
 
-execPath, err := i.Ensure(context.Background(), []src.Source{
-  &fs.ExactVersion{
-    Product: product.Terraform,
-    Version: v0_14_0,
-  },
-  &releases.ExactVersion{
-    Product: product.Terraform,
-    Version: v0_14_0,
-  },
-})
-if err != nil {
-  // process err
-}
+Given that one of the key roles of the CLI/library is integrity checking, you should choose the installation method which involves the same level of integrity checks, and/or perform these checks yourself. `go install` provides only minimal to no integrity checks, depending on exact use. We recommend any of the installation methods documented below.
 
-// run any tests
+#### Homebrew (macOS / Linux)
 
-defer i.Remove()
+[Homebrew](https://brew.sh)
+
+```
+brew install hashicorp/tap/hc-install
 ```
 
-### Install multiple versions
+#### Linux
+
+We support Debian & Ubuntu via apt and RHEL, CentOS, Fedora and Amazon Linux via RPM.
+
+You can follow the instructions in the [Official Packaging Guide](https://www.hashicorp.com/official-packaging-guide) to install the package from the official HashiCorp-maintained repositories. The package name is `hc-install` in all repositories.
+
+#### Other platforms
+
+1. [Download for the latest version](https://releases.hashicorp.com/hc-install/) relevant for your operating system and architecture.
+2. Verify integrity by comparing the SHA256 checksums which are part of the release (called `hc-install_<VERSION>_SHA256SUMS`).
+3. Install it by unzipping it and moving it to a directory included in your system's `PATH`.
+4. Check that you have installed it correctly via `hc-install --version`.
+  You should see the latest version printed to your terminal.
+
+### Usage
 
-```go
-TODO
 ```
+Usage: hc-install install [options] -version <version> <product>
 
-### Install and build multiple versions
-
-```go
-i := NewInstaller()
-
-vc, _ := version.NewConstraint(">= 0.12")
-rv := &releases.Versions{
-  Product:     product.Terraform,
-  Constraints: vc,
-}
-
-versions, err := rv.List(context.Background())
-if err != nil {
-  return err
-}
-versions = append(versions, &build.GitRevision{Ref: "HEAD"})
-
-for _, installableVersion := range versions {
-  execPath, err := i.Ensure(context.Background(), []src.Source{
-    installableVersion,
-  })
-  if err != nil {
-    return err
-  }
-
-  // Do some testing here
-  _ = execPath
-
-  // clean up
-  os.Remove(execPath)
-}
+  This command installs a HashiCorp product.
+  Options:
+    -version  [REQUIRED] Version of product to install.
+    -path     Path to directory where the product will be installed. Defaults
+              to current working directory.
+```
+```
+$ hc-install install -version 1.3.7 terraform
+```
+```
```

Comment:
```suggestion
```

To keep the output in the same block as the command

Reply:
I actually did this on purpose, as the two blocks represent different "syntax" and we could highlight them differently (although we aren't atm) - i.e. the top should be valid shell, but the bottom is text output. It's just semantics though - I don't mind either way! 😄

Reply:
I see; then I would favor removing the `$` in the first block, so it's easily copyable and executable. But feel free to merge anyway, as it's just semantics, as you said.